internal/inspect/detectors/quarto.go

```diff
@@ -42,166 +41,17 @@ func NewQuartoDetector(log logging.Logger) *QuartoDetector {
 	}
 }
 
-type quartoMetadata struct {
-	Title   string `json:"title"`
-	Runtime string `json:"runtime"`
-	Server  any    `json:"server"`
-}
-
-type quartoProjectConfig struct {
```

Comment:
Moved related `quarto inspect` output objects to it's own file.

Reply:
Thanks that was helpful 😄